external/bluetooth/test/bt_unit_test.c

```diff
@@ -0,0 +1,9442 @@
+	{"bt_adapter_is_discovering"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_IS_DISCOVERING},
+	{"bt_adapter_foreach_bonded_device"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_FOREACH_BONDED_DEVICE},
+	{"bt_adapter_foreach_profile_connected_devices"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_FOREACH_PROFILE_CONNECTED_DEVICES},
+	{"bt_adapter_get_bonded_device_info"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_GET_BONDED_DEVICE_INFO},
+	{"bt_adapter_is_service_used"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_IS_SERVICE_USED},
+	{"bt_adapter_get_visibility"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_GET_VISIBILITY},
+	{"bt_adapter_set_visibility"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_SET_VISIBILITY},
+	{"bt_adapter_set_device_discovery_state_changed_cb"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_SET_DEVICE_DISCOVERY_STATE_CHANGED_CB},
+	{"bt_adapter_unset_device_discovery_state_changed_cb"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_UNSET_DEVICE_DISCOVERY_STATE_CHANGED_CB},
+	{"bt_adapter_get_local_oob_data"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_GET_LOCAL_OOB_DATA},
+	{"bt_adapter_set_remote_oob_data"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_SET_REMOTE_OOB_DATA},
+	{"bt_adapter_remove_remote_oob_data"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_REMOVE_REMOTE_OOB_DATA},
+	{"bt_adapter_get_local_oob_ext_data"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_GET_LOCAL_OOB_EXT_DATA},
+	{"bt_adapter_set_remote_oob_ext_data"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_SET_REMOTE_OOB_EXT_DATA},
+	{"bt_adapter_set_visibility_mode_changed_cb"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_SET_VISIBILITY_MODE_CHANGED_CB},
+	{"bt_adapter_unset_visibility_mode_changed_cb"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_UNSET_VISIBILITY_MODE_CHANGED_CB},
+	{"bt_adapter_set_visibility_duration_changed_cb"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_SET_VISIBILITY_DURATION_CHANGED_CB},
+	{"bt_adapter_unset_visibility_duration_changed_cb"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_UNSET_VISIBILITY_DURATION_CHANGED_CB},
+	{"bt_adapter_set_connectable_changed_cb"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_SET_CONNECTABLE_CHANGED_CB},
+	{"bt_adapter_unset_connectable_changed_cb"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_UNSET_CONNECTABLE_CHANGED_CB},
+	{"bt_adapter_get_connectable"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_GET_CONNECTABLE},
+	{"bt_adapter_set_connectable"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_SET_CONNECTABLE},
+	{"bt_adapter_get_version"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_GET_VERSION},
+	{"bt_adapter_get_local_info"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_GET_LOCAL_INFO},
+	{"bt_adapter_set_manufacturer_data_changed_cb"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_SET_MANUFACTURER_DATA_CHANGED_CB},
+	{"bt_adapter_unset_manufacturer_data_changed_cb"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_UNSET_MANUFACTURER_DATA_CHANGED_CB},
+	{"bt_adapter_set_manufacturer_data"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_SET_MANUFACTURER_DATA},
+	{"bt_adapter_set_authentication_req_cb"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_SET_AUTHENTICATION_REQUSET_CB},
+	{"bt_adapter_unset_authentication_req_cb"
+		, BT_UNIT_TEST_FUNCTION_ADAPTER_UNSET_AUTHENTICATION_REQUSET_CB},
```

Comment:
Are you going to have BT test with this file? if not you'd better not to include it

Reply:
I will remove test file.